core-api/src/main/java/org/silverpeas/core/date/period/UndefinedPeriod.java

```diff
@@ -30,8 +30,10 @@
 import java.util.TimeZone;
 
 /**
- * An immutable UNDIFINED period.
+ * An immutable UNDEFINED period.
+ * @deprecated Use {@link org.silverpeas.core.date.Period}
  */
+@Deprecated
 class UndefinedPeriod extends Period {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1133)